### PR TITLE
Next: v0.12

### PR DESCRIPTION
**Pull request by raquo**
Comes with [Airstream upgrades](https://github.com/raquo/Airstream/pull/48) and latest Scala DOM Types

### CODE REVIEW

File: src/main/scala/com/raquo/laminar/keys/CompositeKey.scala [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import com.raquo.airstream.core.Observable
+import com.raquo.laminar.api.Laminar.{MapValueMapper, StringValueMapper}
+import com.raquo.laminar.keys.CompositeKey.CompositeValueMapper
+import com.raquo.laminar.modifiers.{Binder, Setter}
+import com.raquo.laminar.nodes.ReactiveElement
+
+import scala.scalajs.js
+
+// @TODO[Performance] We can eventually use classList for className attribute instead of splitting strings. That needs IE 10+
+// @TODO[Performance] Will string splitting be faster with native JS method? How do we access it?
```
Comment (keynmol) [inline src/main/scala/com/raquo/laminar/keys/CompositeKey.scala:12]: I think

```
import scala.scalajs.js.JSStringOps._
```

puts `<String>.jsSplit(...)` extension method in your scope: https://github.com/scala-js/scala-js/blob/master/library/src/main/scala/scala/scalajs/js/JSStringOps.scala#L162

Comment (raquo) [reply src/main/scala/com/raquo/laminar/keys/CompositeKey.scala]: Nice, thanks!

**Comment by ngbinh** [comment]
@raquo  This PR used to be huge (8K+ lines changes), is there something missing here?

**Comment by ngbinh** [comment]
Oops, sorry. Mistaken with https://github.com/raquo/Airstream/pull/48

**Pull request merged**